### ISSUE TITLE
confused with ignore_mask and true_object_mask

### ISSUE THREAD

**Issue by speeding-motor** [calmisential/YOLOv3_TensorFlow2#12]
```
ignore_mask = best_ious < ignore_threshold
confidence_loss = true_object_mask * self.__binary_crossentropy_keep_dim(true_object_mask, pred_features[..., 4:5], from_logits=True) + \
                              (1 - true_object_mask) * self.__binary_crossentropy_keep_dim(true_object_mask, pred_features[..., 4:5], from_logits=True) * ignore_mask
```
when compute the confidence_loss, here you use the ignore_mask, and only in here, other place all use the true_object_mask, I m really confused with this part...

any reply is appreciate, thanks in advance
@calmisential

**Comment by calmiLovesAI** [comment]
The role of ignore_mask is to ignore some negative samples when calculating the confidence loss.
